frontend/javascripts/oxalis/view/action-bar/download_modal_view.tsx

```diff
@@ -268,10 +278,13 @@ function _DownloadModalView({
     (state: OxalisState) => state.temporaryConfiguration.isMergerModeEnabled,
   );
   const hasVolumeFallback = tracing.volumes.some((volume) => volume.fallbackLayer != null);
+  const isVolumeNDimensional = tracing.volumes.some((tracing) => tracing.additionalAxes.length > 0);
+  const hasVolumes = hasVolumeTracings(tracing);
+  const initialFilesToDownload = hasVolumes ? (isVolumeNDimensional ? "zarr" : "wkw") : "nml";
```

Comment:
I'd rename `initialFilesToDownload` and `filesToDownload` to `[initial]fileFormatToDownload`. otherwise, it sounds like the vars would contain arrays.